app/actors/sufia/grant_edit_to_depositor_actor.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  class GrantEditToDepositorActor < CurationConcerns::Actors::GrantEditToDepositorActor
+    private
+
+      # Overriding the parent class so it doesn't grant access
+      # if mediated deposit is enabled
+      def grant_edit_access
+        super unless mediated_deposit?
```

Comment:
No. The return value of this method is inconsequential https://github.com/projecthydra/curation_concerns/commit/7169243c5d201514dc6204e54369d16c2545b452#diff-e93e2181c149e5670c6afc7cf3f216c7R8